models/topics.models.js

```diff
@@ -1,13 +1,21 @@
 const db =require('../db/connection.js')
+const fs = require('fs').promises
+const path = require('path')
 
+const readEndpoints = () => {
```

Comment:
This may sound a little counterintuitive, but this should be in the controller. The reason for this is because we aren't interacting with the database here.